### PR TITLE
Added mysql & postgres spec with restricted PSA

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
Added mysql & postgres spec with restricted PSA
**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

Verified by adding restricted label to the namespaces & applying the spec files.
Pods are in Running state
```
`[root@ip-10-13-232-124 mysql-backup]# kubectl get ns mysql-latest-psa-rest --show-labels
NAME                    STATUS   AGE     LABELS
mysql-latest-psa-rest   Active   2m47s   kubernetes.io/metadata.name=mysql-latest-psa-rest,pod-security.kubernetes.io/enforce=restricted
[root@ip-10-13-232-124 mysql-backup]# kubectl get pods -n mysql-latest-psa-rest
NAME                         READY   STATUS    RESTARTS   AGE
mysql-558dd79798-9h6nc       1/1     Running   0          2m31s
mysqlslap-5f9c6fbcdc-5gtgk   1/1     Running   0          2m31s
[root@ip-10-13-232-124 mysql-backup]#
[root@ip-10-13-232-124 mysql-backup]#
[root@ip-10-13-232-124 mysql-backup]# kubectl get ns pg1-csi-16 --show-labels
NAME         STATUS   AGE   LABELS
pg1-csi-16   Active   16m   kubernetes.io/metadata.name=pg1-csi-16,pod-security.kubernetes.io/enforce=restricted
[root@ip-10-13-232-124 mysql-backup]# kubectl get pods -n pg1-csi-16
NAME                        READY   STATUS    RESTARTS   AGE
postgres-657d49f65c-c7mzc   1/1     Running   0          15m
[root@ip-10-13-232-124 mysql-backup]#
```

### CODE REVIEW

File: drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/px-mysql-app.yaml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      labels:
+        app: mysql
+    spec:
+      schedulerName: stork
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+        fsGroup: 1010
+      containers:
+      - image: mysql:5.6
```
Comment (vpinisetti-px) [inline drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/px-mysql-app.yaml:47]: @snigdha-px can you try with 8.0 or 8.4 as 5.6 very old version.

File: drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/pxd/px-storage-class.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+##### Portworx storage class
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: mysql-sc
+provisioner: kubernetes.io/portworx-volume
```
Comment (vpinisetti-px) [inline drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/pxd/px-storage-class.yaml:6]: @snigdha-px Its better to change it to CSI driver

File: drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/px-postgres-app.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          - labelSelector:
+              matchExpressions:
+              - key: "app"
+                operator: In
+                values:
+                - postgres
+            topologyKey: "kubernetes.io/hostname"
+      containers:
+      - name: postgres
+        image: postgres:9.5
```
Comment (vpinisetti-px) [inline drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/px-postgres-app.yaml:56]: Same here @snigdha-px . Lets try with `16.0`

File: drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+##### Portworx storage class
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: postgres-sc
+provisioner: kubernetes.io/portworx-volume
```
Comment (vpinisetti-px) [inline drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml:6]: Same here.. change with CSI driver

**Review by vpinisetti-px — CHANGES_REQUESTED** [review]

**Review by kshithijiyer-px — APPROVED** [review]
LGTM, Let's add a screenshot of the running apps if you have tried in on your running cluster.

File: drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml (hunk 4 above)
Comment (snigdha-px) [reply drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml]: done

File: drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/px-postgres-app.yaml (hunk 3 above)
Comment (snigdha-px) [reply drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/px-postgres-app.yaml]: done

File: drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/px-mysql-app.yaml (hunk 1 above)
Comment (snigdha-px) [reply drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/px-mysql-app.yaml]: done

**Comment by snigdha-px** [comment]
> LGTM, Let's add a screenshot of the running apps if you have tried in on your running cluster.

done

File: drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml (hunk 4 above)
Comment (kphalgun-px) [reply drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml]: Caution: Torpedo, by default, uses the `kubernetes.io/portworx-volume` provisioner. We need to pass `CSI_APP_LIST` for it to deploy apps with the `pxd.portworx.com` provisioner. Cc: @sn-px

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Comment by avinashsingh-px** [comment]
By passing TRAVIS Checks, as the agent have issues.

**Pull request merged**